todo_app/tests/test_viewmodel.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from todo_app.Card import Card
+from todo_app.ViewModel import ViewModel 
+from todo_app.data import trello_items
+
+@pytest.fixture
+def board_prep(): 
+    not_started_card = Card("1", "Not Started Test", "Not Started", "test")
+    in_progress_card = Card("2", "In Progress Test", "In Progress", "test")
+    done_card = Card("3", "Done Test", "Done", "test")
+    cards = [not_started_card, in_progress_card, done_card]
+    return ViewModel(cards)
+
+# @pytest.fixture
+# def done_cards_prep():
+#     done_card_1 = Card("3", "Done Test", "Done", "test")
+#     done_card_2 = Card("4", "Done Test", "Done", "test")
+#     done_card_3 = Card("5", "Done Test", "Done", "test")
+#     done_card_4 = Card("6", "Done Test", "Done", "test")
+#     done_card_5 = Card("7", "Done Test", "Done", "test")
+#     done_card_6 = Card("8", "Done Test", "Done", "test")
+#     done_cards = [done_card_1, done_card_2, done_card_3, done_card_4, done_card_5, done_card_6]
+#     return ViewModel(done_cards)
```

Comment:
Try and avoid committing commented out code and get in the habit of using branches if you want to experiment with something.